config/core/configmaps/data-residency.yaml

```diff
@@ -48,6 +48,8 @@ data:
       # clusterDefaults are the defaults to apply to every namespace in the
       # cluster
       clusterDefaults:
+        # messagestoragepolicy.global will set to the Org Policy
```

Comment:
So we want to have: 
1.  Allow all regions, default to global or Org Policy. In code the topic config filed will be nil.
2. No user setting, default to cluster region.
3. Specified regions.

Our discussion result was for 1 using user input `global`, for 2 use `[]`, for 3 use non empty setting.  so global is actually a kind of user input to avoid typing out all possible fields. We can potentially use nil as global, but think that this will be confusing.

Reply:
Fixed as suggested.